code/modules/trade/datums/trade_stations_presets/fs_experimental_factory.dm

```diff
@@ -0,0 +1,25 @@
+/datum/trade_station/fs_experimental_factory
+	name_pool = list("FSTB 'Suoh'" = "Frozen Star Trade Beacon 'Suoh'. This is the experimental weaponry seller, requires a better description because I suck at this, please someone on the PR send help.")
+	markup = 1
+	spawn_probability = 10
+	assortiment = list(
+		"Gunmods" = list(
+			/obj/item/weapon/gun_upgrade/trigger/dangerzone,
+			/obj/item/weapon/gun_upgrade/trigger/cop_block,
+			/obj/item/weapon/gun_upgrade/mechanism/weintraub,
+			/obj/item/weapon/gun_upgrade/scope/watchman
+		),
+		"Grenades" = list(
+			/obj/item/weapon/grenade/anti_photon,
+			/obj/item/weapon/grenade/empgrenade,
+			/obj/item/weapon/grenade/chem_grenade/incendiary
```

Comment:
Isn't there already a station that sells this grenade?

Reply:
If you mean the incendiary, it was moved here, though . This is the normal emp grenade (the stronger version, not the weak one). And I just noticed I forgot to remove the stronger one of Kaida.